### ISSUE TITLE
touchscreen icons don't fit to their buttons

### ISSUE THREAD

**Issue by carandraug** [MicronOxford/cockpit#717]
The icons in the touchscreen buttons fit nicely inside the buttons on Windows only. In GTK3 and macOS they do not which looks odd.  This how some of them look in GTK3:

![touchscreen-icons](https://user-images.githubusercontent.com/916140/109867006-eaeaef00-7c5d-11eb-8874-52f6208428ca.png)

**Comment by iandobbie** [comment]
I tried changing the button size to 56x56 rather than 48x48 and it all looked much better on a mac, commit https://github.com/iandobbie/cockpit/commit/e323c7784312bf97de514393fede5f2779c57487

**Comment by carandraug** [comment]
Is there no way to avoid hardcoded button sizes?

**Comment by iandobbie** [comment]
I think with no size the button should be created exactly the right size to fit its contents.

**Comment by iandobbie** [comment]
Yes it does, but not as you might expect. Button are then too wide.
![image](https://user-images.githubusercontent.com/5451909/110377947-eb63fb00-804c-11eb-8320-4985ae5a7eeb.png)

**Comment by iandobbie** [comment]
Can be improved on with a call to 
self.SetWindowStyleFlag(wx.BU_EXACTFIT)
![image](https://user-images.githubusercontent.com/5451909/110380358-e6547b00-804f-11eb-9ffc-7a92c4637f15.png)

**Comment by iandobbie** [comment]
Still seems to have more room on the right than the left for some reason.
